### ISSUE TITLE
[Bug] PC failing to upload to bestiary.

### ISSUE THREAD

**Issue by Iankid** [WBHarry/pf2e-bestiary-tracking#107]
Not sure why, but one PC our of my party of 6 is failing to upload with the below error in the console.
 I attempted to recreate their sheet and ran into the same issue.
    
 I'm also providing the PC's sheet in question.
  
[fvtt-Actor-akira-(剣)-scTOUv9HQqgjkL2X.json](https://github.com/user-attachments/files/17038360/fvtt-Actor-akira-.-scTOUv9HQqgjkL2X.json)

```Uncaught (in promise) TypeError: item is undefined
[Detected 1 package: pf2e-bestiary-tracking(1.0.9)]
    getPCCreatureData https://assets.forge-vtt.com/bazaar/modules/pf2e-bestiary-tracking/1.0.9/BestiaryTracking.js:3209
    getCreatureData https://assets.forge-vtt.com/bazaar/modules/pf2e-bestiary-tracking/1.0.9/BestiaryTracking.js:2640
    getNPCData https://assets.forge-vtt.com/bazaar/modules/pf2e-bestiary-tracking/1.0.9/BestiaryTracking.js:3606
    _onDrop https://assets.forge-vtt.com/bazaar/modules/pf2e-bestiary-tracking/1.0.9/BestiaryTracking.js:15080`

**Comment by WBHarry** [comment]
I've made a little change on my dev branch that fixes this. I tried uploading the sheet you provided, and it works fine there.
So it'll come out with the next version 🤖 👍

**Comment by WBHarry** [comment]
I just released v1.0.10 that fixes this. I could successfully import the character
